pkg/plugin/plugin_config.go

```diff
@@ -0,0 +1,24 @@
+// Copyright 2018-2021 CERN
```

Comment:
Rename the file

Reply:
Does it make sense to rename this to `plugin.go` and rename the current `plugin.go` to `loader.go`? because that's what the current `plugin.go` is doing! Not too sure though.